views/calendar/calendar.js

```diff
@@ -159,22 +163,11 @@ export default class CalendarView extends React.Component {
     }
 
     if (this.state.error) {
-      return <Text>{this.state.error}</Text>
+      return <NoticeView text={this.state.error} />
     }
 
-    if (this.state.noEvents) {
-      return (
-        <View style={{
-          flex: 1,
-          justifyContent: 'center',
-          alignItems: 'center',
-          backgroundColor: '#ffffff',
-        }}>
-          <Text>
-            No events.
-          </Text>
-        </View>
-      )
+    if (!this.state.events.getRowCount()) {
+      return <NoticeView text='No events.' />
```

Comment:
This is precisely why we have NoticeView– to allow some text to take up an entire screen and be centered.